package.json

```diff
@@ -17,12 +17,14 @@
     "node": ">=6.8.0"
   },
   "dependencies": {
-    "blake2": "^1.4.0",
+    "blakejs": "github:dcposch/blakejs",
```

Comment:
Why switch dependency, and why not use the published npm module?

Reply:
Because a lot of people have problems with compiling blake2 with `node-gyp`, especially on Windows. Enter `blakejs` - a pure JavaScript impl of blake2.